be/test/formats/orc/orc_test_util/MemoryInputStream.cc

```diff
@@ -0,0 +1,27 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2022-present, StarRocks Inc.
```

Comment:
There are memory input stream implementation in orc. Don't copy it.

Reply:
I can't use MemoryInputStream directly, it's in `/apache-orc` test folder, and will not be included by StarRocks.

Reply:
Yes, put orc lazy load test in apache-orc directory.